rerun_py/rerun_sdk/rerun/sinks.py

```diff
@@ -32,16 +32,44 @@ def is_recording_enabled(recording: RecordingStream | None) -> bool:
 LogSinkLike = Union[GrpcSink, FileSink]
 
 
-def _tee(
+def attach_sinks(
     *sinks: LogSinkLike,
     default_blueprint: BlueprintLike | None = None,
     recording: RecordingStream | None = None,
 ) -> None:
     """
     Stream data to multiple different sinks.
```

Comment:
here and on `recording_stream`:

what happens to any data logged so far, will those new sinks catch up (I reckon no).
I think this is important to clarify.

the interaction of `spawn=True` and `attach_sinks` is also worth pointing out on `rr.init`'s documentation

Reply:
words are hard. it still seems a bit complicated but i've probably spent enough time trying to reword it, so i'll leave it as is in the latest commit and merge. the fact that it's not "easily" documentable suggests to me that what we have is overly complex, and we should head towards "set sinks" as the only thing asap.